### ISSUE TITLE
provide tzinfo attribute on cftime.datetime objects

### ISSUE THREAD

**Issue by rabernat** [Unidata/cftime#208]
Cftime datetimes are mostly duck-compatible with standard python datetimes.  However, they lack the `tzinfo` attribute. 

```python
import datetime
d = datetime.datetime(2000, 1, 1)
assert d.tzinfo is None

import cftime
c = cftime.datetime(2000, 1, 1)
assert c.tzinfo is None
# -> AttributeError: 'cftime._cftime.datetime' object has no attribute 'tzinfo'
```

This caused a bug recently in pystack: https://github.com/stac-utils/pystac/pull/238. I notice it also caused an internal bug in cftime (#126), which was fixed with #127 by replacing a reference to `.tzinfo` with a `getattr` call (similar to my workaround in https://github.com/stac-utils/pystac/pull/238).

Rather than patching all the downstream libraries that might want to work with cftime datetimes (which might not realize they are not regular datetimes) to avoid assuming this attribute exist, wouldn't it be easier to provide a `.tzinfo` attribute on cftime dates?

Thanks for considering my suggestion.

**Comment by jswhit** [comment]
What do you suggest?  cftime.datetime.tzinfo = None?

**Comment by rabernat** [comment]
Yes that would be consistent with datetime.

**Comment by jswhit** [comment]
closed by PR #209

**Comment by rabernat** [comment]
Thank you so much!
